Examples/Io/Root/include/ActsExamples/Io/Root/RootParticleWriter.hpp

```diff
@@ -106,6 +115,27 @@ class RootParticleWriter final : public WriterT<SimParticleContainer> {
   std::vector<float> m_phi;
   /// Transverse momentum in GeV.
   std::vector<float> m_pt;
+  /// Polar angle.
+  std::vector<float> m_theta;
+  /// Charge over momentum in e.GeV^-1.
+  std::vector<float> m_qop;
+
+  /// Add perigee prefix to the above parameters
+  /// if m_cfg.writeHelixParameters is true.
+  std::vector<float> m_perigee_phi;
+  std::vector<float> m_perigee_theta;
+  std::vector<float> m_perigee_qop;
+  std::vector<float> m_perigee_p;
+  std::vector<float> m_perigee_px;
+  std::vector<float> m_perigee_py;
+  std::vector<float> m_perigee_pz;
+  std::vector<float> m_perigee_eta;
+  std::vector<float> m_perigee_pt;
+  /// Transverse impact parameter in mm.
+  std::vector<float> m_perigee_d0;
+  /// Longitudinal impact parameter in mm.
+  std::vector<float> m_perigee_z0;
```

Comment:
nitpick: after the `m_` the convention is to use snakeCase

Reply:
Done!